src/officehours_ui/templates/registration/login.html

```diff
@@ -19,12 +19,12 @@
             <form method="post" action="{% url 'login' %}">
                 {% csrf_token %}
                 <div class="form-row">
-                    <div class="form-group col-md-6">
-                        <label for="username">Username</label>
+                    <div class="col-md-6 mb-3">
+                        <label class="mb-1" for="username">Username</label>
                         <input class="form-control" name="username" type="text" placeholder="Username" autofocus autocapitalize="none" autocomplete="username" maxlength="150" required id="username">
                     </div>
-                    <div class="form-group col-md-6">
-                        <label for="password">Password</label>
+                    <div class="col-md-6 mb-3">
+                        <label class="mb-1" for="password">Password</label>
```

Comment:
Was these changes adaptation to new version of Boostrap or making things look better?

Reply:
I think a little of both. https://getbootstrap.com/docs/5.3/migration/#forms-1 You can see that `form-group` is no longer recommended (maybe not supported either?). Just FYI, these login forms are not used in production, where we use SSO.